crates/bevy_ecs/src/bundle.rs

```diff
@@ -22,6 +22,8 @@ use std::{any::TypeId, collections::HashMap};
 /// particularly useful for spawning multiple empty entities by using
 /// [`Commands::spawn_batch`](crate::system::Commands::spawn_batch).
 ///
+/// To use a bundle dynamically, use [`Box<dyn ApplicableBundle>`](`ApplicableBundle`)
```

Comment:
```suggestion
/// This trait is not object-safe: to create a bundle with a flexible set of components, use [`Box<dyn ApplicableBundle>`](`ApplicableBundle`)
```

Reply:
Maybe? My only concern here is that the returned item isn't 'a bundle' as such (in this context, a bundle means an implementation of `Bundle`. This feature explicitely disallows nesting within `Bundle`s to keep some of the useful properties `Bundle` currently has.